orm/internal/codegen/index.go

```diff
@@ -5,19 +5,18 @@ import (
 	"strings"
 
 	"github.com/iancoleman/strcase"
-	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
 func (t tableGen) genIndexKeys() {
```

Comment:
Let's not change anything in the codegen in a linting PR, please.
Changing it to any makes the generated code require a specific minimum go version, while it wasn't the case before.